executor/src/witgen/jit/block_machine_processor.rs

```diff
@@ -381,11 +230,12 @@ params[2] = Add::c[0];"
             .err()
             .unwrap();
         assert!(err_str
-            .contains("Unable to derive algorithm to compute output value \"Unconstrained::c\""));
+            .contains("The following variables or values are still missing: Unconstrained::c"));
     }
 
     #[test]
-    #[should_panic = "Block machine shape does not allow stacking"]
+    //#[should_panic = "Column NotStackable::a is not stackable in a 1-row block"]
```

Comment:
```suggestion
```

Reply:
The idea is that we change the expectation once the stackability check panics again.